Makefile

```diff
@@ -66,7 +66,7 @@ TEST_E2E_PKGS = $(shell $(GO) list ./... | grep 'github.com/3scale/3scale-operat
 ENVTEST_ASSETS_DIR=$(PROJECT_PATH)/testbin
 test-e2e: generate fmt vet manifests
 	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
```

Comment:
`controller-runtime v0.6.3` is being used when compiling the operator. 

Why should we change the version?

Reply:
`controller-runtime v0.6.3` does not contain ppc64le compatible `kubebuilder-tools`.
Bumping up the `controller-runtime v0.8.0` will allow `test-e2e` to be executed on ppc64le as well.